1678. Goal Parser Interpretation

```diff
@@ -0,0 +1,19 @@
+class Solution(object):
+    def interpret(self, command):
+        """
+        :type command: str
+        :rtype: str
+        """
+        res = ""
+        i = 0
+        while i < len(command):
+            if command[i] == "G":
+                res += "G"
+                i += 1
+            elif command[i] == "(" and command[i+1] == ")":
```

Comment:
this code probably passed all leetcode tests and probably is fine, but if i is len(command)-1, aka the end of the list, won't this cause a segfault if u do i+1 here?

Reply:
*out of bounds error

Reply:
yeah probbaly should have, but I think the leetcode question in general has protection here, but something to keep in mind during interviews